test/network_test_lacp_properties.ml

```diff
@@ -51,6 +51,7 @@ module OVS_Cli_test = struct
   include Ovs.Cli
   let vsctl_output = ref []
   let vsctl ?log args =
+    ignore(log);
```

Comment:
Perhaps add a comment here, or someone will delete it :)

Reply:
Indeed. What does this do and why does it require parentheses?

Reply:
Good point on the parentheses - I've obviously not been writing enough OCaml recently :) I'll fix that.

Without this, the tests don't build - dune complains about the unused `log` argument. The argument can't go away entirely though, or the `OVS_Cli_test` module won't conform to the `Ovs.Cli` interface.

Reply:
Use:
```
let vsctl ?log:_ args =
```

Reply:
Done :)